test/test_helper.rb

```diff
@@ -197,6 +197,12 @@ def process(...)
       super
     end
   end
+
+  def verified_sign_in_as(user)
```

Comment:
Is this in the right place?

Reply:
Where is `sign_in_as` defined? it seems not reachable directly from test help (just including test helper will not expose this method). Seems it is defined as needed in tests itself. For now would it be possible to improve the one used in related test files and extend with like `verified: true` or `verified: 10.minutes.from_now`.

```
[retro@retro  rubygems.org (master $%=)]❤ grep -r "def sign_in_as" test/
test/integration/owner_test.rb:  def sign_in_as(user)
test/integration/rack_attack_test.rb:  def sign_in_as(user)
test/system/avo/events_test.rb:  def sign_in_as(user)
test/system/avo/oidc_api_key_roles_test.rb:  def sign_in_as(user)
test/system/avo/users_test.rb:  def sign_in_as(user)
test/system/avo/web_hooks_test.rb:  def sign_in_as(user)
test/system/avo/rubygems_test.rb:  def sign_in_as(user)
test/system/avo/versions_test.rb:  def sign_in_as(user)
test/system/avo/sendgrid_events_test.rb:  def sign_in_as(user)
test/system/avo/manual_changes_test.rb:  def sign_in_as(user)
test/system/avo/oidc_providers_test.rb:  def sign_in_as(user)
test/system/maintenance_tasks_test.rb:  def sign_in_as(user)
```

Reply:
And unify `sign_in_as` methods later (in separate PR, happy to help). I think we will need 2 helpers, one for "clicking tests" and for "unit tests".

Reply:
Created issue #4791.